### DISCUSSION TITLE
Seeking help with item selection and keeping the item selected...

### DISCUSSION THREAD

**Discussion by ishaisagi** [microsoft-search/pnp-modern-search#3800] [General]
Hi all, I hope someone can help.
I have two search results web parts (not 100% sure what version is installed as I am not admin).
I have configured one to allow item selection and filter then other when selected which works great. The issue is, when the user clicks anywhere inside the other web part (for example to copy some text), the selection gets cancelled and the content disappear.
I put  "data-selection-disabled" as an attribute on all tags in the receiving web part to no avail.

Does anyone know how to make this work?

thanks in advance.

**Comment by kasperbolarsen** [comment]
Unless your version is very old, the version number should be visible on the last page of the web part configuration pane.

**Reply by ishaisagi** [reply]
Version: 4.6.1
does that matter?

**Reply by kasperbolarsen** [reply]
Very much, as we can only support the current version ( 4.12.2).

**Comment by ishaisagi** [comment]
So that I can build a business case for the upgrade – can you confirm that this is possible in the latest version?
From my research, the select disable only works for elements inside the web part that enables selection, not when they are applied in a web part that sits next to it (the receiving web part).

Sadly I cannot use javascript on that page (no way to inject it) or I would solve it in another way.

**Reply by kasperbolarsen** [reply]
I can only recommend that you ask your internal IT to set up a site collection where you are the Site Collection admin.  On that site they should enable a site collection level App catalog,  and now you can upgrade the PnP Modern Search solution yourself. 
This will enable you to test out all the options within changing any thing on the tenant. 

An even better approach would be to get a test tenant like here https://www.google.com/amp/s/office365itpros.com/2021/04/15/test-microsoft-365-tenant/amp/

**Reply by patrikhellgren** [reply]
@ishaisagi I can confirm that there has been at least two updates to item selection handling in the details list layout since that version, #3568 and #3810. Both are included in version 4.12.2. I have also done a quick test of your scenario and the selection is not cleared when clicking in the other webpart.

**Comment by ishaisagi** [comment]
Patrick you are a champion!
That gives me a good argument for the upgrade. Hope it won't take too long

Kasper, thanks for responding. I understand your thinking but if I could ask for a side site with an app store I would have. I guess I could have tested on my personal tenant, but I was hoping for someone's insight about why it's not working in the first place.



Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
